backend/cfg/eliminate_useless_edges.ml

```diff
@@ -40,7 +40,14 @@
    - its prececessors are set to empty;
    - (its other fields are left unchanged).
 
-   As a consequence, `b2` becomes dead. *)
+   As a consequence, `b2` becomes dead.
+
+   Note: by taking the "union" of the `exns`, `can_raise`, `can_raise_interproc`,
+   and `can_raise_interproc` fields, we are losing a bit of precision as to
```

Comment:
`can_raise_interproc` appears twice?
It is worth mentioning that it can affect precision of some backward analysis, like liveness (as you pointed out offline).

Reply:
Another case of bad copy/paste, thanks.